.github/CODEOWNERS

```diff
@@ -7,8 +7,8 @@
 ############################
 ##### Smart contracts ######
 ############################
-/contracts                             @AlfredoG87 @ebadiere @georgi-l95 @Ivo-Yankov @Nana-EC @natanasow @hashgraph/hedera-smart-contracts
-/test                                  @AlfredoG87 @attest08 @ebadiere @georgi-l95 @Ivo-Yankov @natanasow @yiliev0 @hashgraph/hedera-smart-contracts
+/contracts                             @hashgraph/hedera-smart-contracts @georgi-l95 @Ivo-Yankov @natanasow
```

Comment:
QQ: Does @hashgraph/hedera-smart-contracts means it would include everyones from @hashgraph/hedera-smart-contracts team?